app/src/main/java/org/simple/clinic/home/patients/PatientsScreen.kt

```diff
@@ -93,10 +94,17 @@ open class PatientsScreen(context: Context, attrs: AttributeSet) : RelativeLayou
         controller = controller,
         screenDestroys = RxView.detaches(this).map { ScreenDestroyed() }
     )
-
     illustrationImageView.setImageResource(illustrationResourceId())
   }
 
+  private fun showIllustration(@IdRes viewId: Int) {
+    illustrationFlipper.apply {
+      val indexOfChildId = indexOfChildId(viewId)
```

Comment:
Can this be,
```kotlin
illustrationFlipper.apply { 
  displayedChild = indexOfChildId(viewId)
}
```

Reply:
I actually added this to avoid multiple calls to updated `displayedChild`.
But I realized that this will only happen once at the time of creating this view, so its fine.

Will make this change.